packages/patternfly-4/react-catalog-view-extension/sass/patternfly-react-extensions/_catalog-tile.scss

```diff
@@ -0,0 +1,76 @@
+.ws-preview {
+  .catalog-tile-pf {
```

Comment:
Is this file for an example? I don't see the file used with an example, but noted the `ws-preview` selector we typically use there.

Reply:
Good catch, I'll fix this later today.